configure.ac

```diff
@@ -2296,6 +2304,21 @@ else
   SKIP_CONFIG_GTEST=1
 fi
 
+# Google Breakpad
+if test "$configure_breakpad" = "yes"; then
+  AC_MSG_NOTICE($breakpad_enabled)
+  final_message="$final_message\n  Google Breakpad Configured:\tYes"
+  USE_BREAKPAD=1
+  PKG_CHECK_MODULES([BREAKPAD],    [breakpad-client],
+    [INCLUDES="$INCLUDES $BREAKPAD_CFLAGS"; LIBS="$LIBS $BREAKPAD_LIBS"],
+    AC_MSG_ERROR($missing_library))
```

Comment:
please:
AC_MSG_ERROR("breakpad-client not found").

and you could move the enabled notice and final_message down after the ac_define.